pack/Km.json

```diff
@@ -97,6 +136,46 @@
         "traits": "",
         "type_code": "event"
     },
+    {
+        "code": "08055",
+        "cost": 5,
+        "deck_limit": 3,
+        "faction_code": "martell",
+        "flavor": "\"I was weaned on venom, Dalt. Any viper takes a bite of me will rue it.\"",
+        "illustrator": "Preston Stone",
+        "is_intrigue": true,
+        "is_loyal": false,
+        "is_military": true,
+        "is_power": false,
+        "is_unique": true,
+        "name": "Darkstar",
+        "octgn_id": null,
+        "pack_code": "Km",
+        "position": 55,
+        "quantity": 3,
+        "strength": 4,
+        "text": "Stealth.\n<b>Reaction:</b> After Darkstar bypasses a character using stealth, that character loses a challenge icon of your choice until the end of the phase.",
+        "traits": "House Dayne. Knight.",
+        "type_code": "character"
+    },
+    {
+        "code": "08056",
+        "cost": 1,
+        "deck_limit": 3,
+        "faction_code": "martell",
+        "flavor": "The sand steeds of Dorne were swift and tireless, and would keep going for long leagues after other horses had given out, but even such as they could not run dry",
+        "illustrator": "Smirtouille",
+        "is_loyal": false,
+        "is_unique": false,
+        "name": "Sand Steed",
+        "octgn_id": "",
```

Comment:
Needs to be `null` until it's filled in with the final value, this currently fails validation.